mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -682,8 +682,9 @@ The `config` block supports:
 * `resilience_mode` -
   (Optional, Cloud Composer 2.1.15 or newer only)
   The resilience mode states whether high resilience is enabled for 
-  the environment or not. Value for resilience mode is `HIGH_RESILIENCE`.
-  If unspecified, defaults to standard resilience.
+  the environment or not. Values for resilience mode are `HIGH_RESILIENCE` 
+  for high resilience and `RESILIENCE_MODE_UNSPECIFIED` for standard
```

Comment:
This should be STANDARD_RESILIENCE to match the validate func on the schema

Reply:
Done.